omnibus/Gemfile

```diff
@@ -5,7 +5,6 @@ gem "omnibus-software", git: "https://github.com/chef/omnibus-software.git", bra
 gem "artifactory"
 
 gem "pedump"
-gem "bcrypt_pbkdf", ">= 1.1.0.rc1"
```

Comment:
We get this through `chef` anyway, right?

Reply:
yep the deps we need for the build are all via chef in the gemfile.lock in omnibus.